tuf/client/updater.py

```diff
@@ -3241,6 +3248,29 @@ def download_target(self, target, destination_directory,
         to ensure snapshot consistency).
         Default is True.
 
+     custom_download_handler:
+        A user provided function performing the actual target file download.
+        In order to comply with the TUF specification, the function implementation
+        should match the following description:
+
+        def download_handler_func(url)
+          <Purpose>
+            Given the 'url' of the desired file,
+            open a connection to 'url', download it, and return the contents
+            of the file.
+
+          <Arguments>
+            url:
+              A URL string that represents the location of the file.
+
+          <Side Effects>
+            A temprorary file object is created to store the contents of 'url'.
+
+          <Returns>
+            A temporary file object that points to the contents of 'url'.
```

Comment:
This description of the return value feels a bit awkward, but I see you've copied it from `tuf.download.safe_download()`